projects/js-packages/publicize-components/src/components/form/index.js

```diff
@@ -67,13 +67,15 @@ export default function PublicizeForm( {
 	const hasInstagramConnection = connections.some(
 		connection => connection.service_name === 'instagram-business'
 	);
-	const [ shouldShowInstagramNotice, setShouldShowInstagramNotice ] = useState(
-		! hasInstagramConnection && isInstagramConnectionSupported
-	);
```

Comment:
It's always good to avoid `useState` for the initial values that can change. The state can get quickly out of sync.